### ISSUE TITLE
Need For Speed - Shift SHIFT

### ISSUE THREAD

**Issue by huchao2007** [EKA2L1/EKA2L1#416]
https://user-images.githubusercontent.com/78587142/172468396-3f7e8105-61fe-49de-a54e-6b9ab5819e42.mp4

**Comment by MTPxD** [comment]
Textures are missing in NFS Shift?

**Comment by pent0** [comment]
Yeah, it has been confirmed to be fixed for a while on Discord. I will close it!
